src/Magento/FunctionalTestingFramework/Test/Config/Converter/Dom/Flat.php

```diff
@@ -124,6 +124,12 @@ public function convertXml(\DOMNode $source, $basePath = '')
             ) {
                 $value = $node->nodeValue;
                 break;
+            } elseif ($node->nodeType == XML_COMMENT_NODE) {
+                $uniqid = uniqid(str_replace("#", "", $node->nodeName));
```

Comment:
Is there anyway we can change this `"#"` to incorporate the constant in `ActionObject.php` or vice versa (keep constant in this class, actionObject uses it instead)?

Having the `#comment` constant to match to this line's output feels disjointed, and one can go out of sync with the other easily.